caffe2/utils/threadpool/pthreadpool_impl.cc

```diff
@@ -28,7 +28,20 @@ void pthreadpool_compute_1d(
 }
 
 size_t pthreadpool_get_threads_count(pthreadpool_t threadpool) {
-  return reinterpret_cast<caffe2::ThreadPool*>(threadpool)->getNumThreads();
+  if (threadpool) {
```

Comment:
Can this be pulled into a separate diff that we can land now?

Reply:
Here: https://github.com/pytorch/pytorch/pull/34087. Added you to it.